### PR TITLE
Set up React Router

### PR DESCRIPTION
**Pull request by ghost**
Please, review the code. I set up React Router for the homepage, cart and also an error page. I've commented out some code for now. I will sort it out later. Have fun! :smile:

### CODE REVIEW

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const HomePage = () => {
+  return (
+    <div>
+      <h2>Welcome to SecondhandMovies.com</h2>
```
Comment (asjoss) [inline src/pages/HomePage.jsx:6]: On the Slack channel we came up with either Caaakewalk DVDs or Re-run DVD's for website name

**Review by asjoss — APPROVED** [review]
All happy with this; so I approve merging the changes

**Review by KerryCX — APPROVED** [review]
Good work on the routing Abul. Go ahead and merge.

File: src/pages/HomePage.jsx (hunk 1 above)
Comment (KerryCX) [reply src/pages/HomePage.jsx:6]: Caaakewalk DVDs gets my vote

**Pull request merged**